### PR TITLE
gnrc: Set address valid lifetime to UINT32_MAX for border router scenario

### PR DESCRIPTION
**Pull request by gebart**
http://riot-os.org/api/structgnrc__ipv6__netif__addr__t.html#a54a9cd73d1bb5dfa39b7d79de51bd46a

>If it is UINT32_MAX the lifetime is infinite.
Definition at line 251 of file ipv6/netif.h.

I think the original intention of the 0xFFFF literal was to make it permanent on border router configurations.

### CODE REVIEW

**Comment by authmillenon** [comment]
Japp, ACK and go when Travis is happy.

**Comment by gebart** [comment]
I found another problem, there's nowhere an already set valid_timeout timer will be removed from xtimer if an address entry is reused

**Comment by authmillenon** [comment]
Can you rephrase that? I'm not really getting what you mean. Is the valid_timeout not removed when an address is removed from the interface?

**Comment by gebart** [comment]
@authmillenon yes, from what I can see in the code the xtimer_t valid_timeout is being queued at 
https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c#L793
but never removed anywhere, so after it has been queued it will only be postponed by the same line on the next call, but not removed if valid is changed to UINT32_MAX.

I don't know if that is related to the problems I was seeing or not, but it would make sense to unqueue the timer if the valid lifetime is changed to infinite.

**Comment by authmillenon** [comment]
Yes, definitely!
